incubator/cassandra/values.yaml

```diff
@@ -0,0 +1,58 @@
+## Cassandra image version
+## ref: https://hub.docker.com/r/library/cassandra/
+image:
+  repo: "cassandra"
+  tag: "latest"
+  pullPolicy: IfNotPresent
+
+## Specify a service type
+## ref: http://kubernetes.io/docs/user-guide/services/
+service:
+  type: ClusterIP
+
+## Persist data to a persitent volume
+persistence:
+  enabled: true
+  storageClass: generic
```

Comment:
this should be commented out, e.g. https://github.com/kubernetes/charts/blob/master/stable/chronograf/values.yaml#L15